pkg/features/kube_features.go

```diff
@@ -625,7 +625,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	CSIInlineVolume:                {Default: true, PreRelease: featuregate.Beta},
 	RuntimeClass:                   {Default: true, PreRelease: featuregate.Beta},
 	NodeLease:                      {Default: true, PreRelease: featuregate.GA, LockToDefault: true},
-	SCTPSupport:                    {Default: false, PreRelease: featuregate.Alpha},
+	SCTPSupport:                    {Default: true, PreRelease: featuregate.Beta},
```

Comment:
This depends on the other PRs with the tests and the prow job

Reply:
Fixed. Removed from this PR.